modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPConfigurationEntryLocalServiceImpl.java

```diff
@@ -120,6 +120,11 @@ public CPConfigurationEntry addCPConfigurationEntry(
 
 		JSONSerializer jsonSerializer = _jsonFactory.createJSONSerializer();
 
+		jsonSerializer.exclude(
+			"CPConfigurationEntryId", "CPConfigurationListId", "ctCollectionId",
+			"companyId", "externalReferenceCode", "mvccVersion", "uuid",
```

Comment:
I think we need to sort `ctCollectionId` that should go after `companyId` (like in the test) and the `uuid` at the end